website/src/pages/code-search.tsx

```diff
@@ -35,21 +35,21 @@ export const CodeSearchPage: FunctionComponent<PageProps> = props => (
                                 data-button-style={buttonStyle.primary}
                                 data-button-location={buttonLocation.hero}
                                 data-button-type="cta"
```

Comment:
Just wanted to note that the data attributes work like this in the current repo with the Gatsby Link component. But in the replatform repo, the data attributes need to be put on the child `a` element because they aren't passed with the Next Link component.

cc: @st0nebraker

Reply:
Roger, @bretthayes! I'll make a note to do so in the porting issue.